lib/praxis/docs/generator.rb

```diff
@@ -85,9 +88,9 @@ def collect_reachable_types( data, reachable_types )
         when Hash
           if data.key?(:type) && data[:type].kind_of?(Hash) && ( [:id,:name,:family] - data[:type].keys ).empty?
             type_id = data[:type][:id]
-            unless type_id.nil?
+            unless type_id.nil? || type_id == Praxis::SimpleMediaType.id #SimpleTypes shouldn't be collected
```

Comment:
The actual error raised is that it's not an `Attributor::Type`, why not check that here instead of whether it's a `SimpleMediaType`?